plugins/source/oracle/resources/services/compute/volume_attachments.go

```diff
@@ -0,0 +1,102 @@
+package compute
+
+import (
+	"context"
+
+	"github.com/cloudquery/cloudquery/plugins/source/oracle/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+	"github.com/oracle/oci-go-sdk/v65/common"
+	"github.com/oracle/oci-go-sdk/v65/core"
+)
+
+type VolumeAttachment struct {
+	// Both core.ParavirtualizedVolumeAttachment & core.EmulatedVolumeAttachment
```

Comment:
Based on the docs it seems `EmulatedVolumeAttachment` is not supported?
See https://pkg.go.dev/github.com/oracle/oci-go-sdk/core#ComputeClient.ListVolumeAttachments
![image](https://github.com/cloudquery/cloudquery/assets/26760571/259b7b74-ee4f-4a9b-8070-b062ef967263)

Not sure we can do anything about it but maybe update the comment?

Reply:
The one missing has just the same fields, so I can either update the comment or leave as is

Reply:
Simplified in c3b6fc0da36df4ba2a39a3da5b4d62bf48264cd5